ui/app/templates/components/secret-list/item.hbs

```diff
@@ -45,7 +45,7 @@
                       </button>
                     </li>
                   {{else}}
-                    {{#if @item.canRead}}
+                    {{#if (or @item.canReadSecretData @item.canRead)}}
```

Comment:
need both because KV has two endpoints to check where other secret-engines only have the one and use canRead. (same with canEdit and canDelete)

Reply:
Yep, that's correct.